src/Uno.UI/Microsoft/UI/Xaml/Controls/Expander/Expander.cs

```diff
@@ -80,8 +83,13 @@ protected override void OnApplyTemplate()
 
 			if (GetTemplateChild<Border>(c_expanderContentClip) is Border expanderContentClip)
 			{
-				var visual = ElementCompositionPreview.GetElementVisual(expanderContentClip);
-				visual.Clip = visual.Compositor.CreateInsetClip();
+				// Uno Doc: The Composition clipping APIs are currently unsupported,
```

Comment:
Please use `// TODO Uno specific:` and wrap in `#if HAS_UNO` here and below, so that it is clear that this is something that should be adjusted in the future and also to know that this must not be reverted when changes are made to this ported file.

Reply:
Thanks Martin. I'll try to get to these changes later this week.

Reply:
@MartinZikmund Added those preprocessor directives and tweaked the inline comments as per your feedback.